react-native/app.config.js

```diff
@@ -0,0 +1,19 @@
+const path = require("path");
+
+module.exports = {
+  expo: {
+    name: "example",
+    slug: "example",
+    version: "1.0.0",
+    orientation: "portrait",
+    userInterfaceStyle: "light",
+    splash: {
+      resizeMode: "contain",
+      backgroundColor: "#ffffff",
+    },
+    assetBundlePatterns: ["**/*"],
+    plugins: [
+      path.resolve(__dirname, "./src/plugins/withReactNativeWebview.js"),
```

Comment:
Added this in order to support Native Modules. However, after exploring a different Web View version in `@thanksjs/react-native-webview`, we may be able to get rid of this.

As you said in the other PR @theKashey, a 'real' RN developer should already know this.